integration_tests/plugins/test_go_plugin.py

```diff
@@ -36,6 +39,25 @@ def test_stage_go_plugin(self):
     def test_building_multiple_main_packages(self):
         self.run_snapcraft('stage', 'go-with-multiple-main-packages')
 
+    def test_building_multiple_main_packages_without_go_packages(self):
+        self.copy_project_to_cwd('go-with-multiple-main-packages')
+
+        for bin in ['main1', 'main2', 'main3']:
+            self.assertThat(os.path.join('stage', 'bin', bin), FileExists())
```

Comment:
You are checking that the stage dir has contents but you haven't called the snapcraft command yet.
`testtools.matchers._impl.MismatchError: stage/bin/main1 does not exist.`